src/botPage/view/logger.js

```diff
@@ -70,8 +70,6 @@ const notifyError = error => {
     if (trackJs && isProduction()) {
         trackJs.console.log(error);
         trackJs.track(code || error.name);
-    } else {
```

Comment:
this else is not necessary because  we get the error at line 7

Reply:
You're right. 👍